pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1683,6 +1685,68 @@
   yw_page_top_heading: "Young Women in Computer Science"
   yw_page_top_desc: "Become a CS Ambassador and be a part of closing the gender gap in computer science."
 
+  lms_skinny_banner_heading: "Discover our new LMS integrations!"
+  lms_skinny_banner_desc: "We've added new LMS integrations for %{canvas} and %{schoology}! Learn how to get started!"
```

Comment:
Non-blocking suggestion: move these into the `lms_page` namespace under a `skinny_banner` section to keep everything contained.

Reply:
I was curious about that- I imagined the banner as separate but perhaps it doesn't matter. I'll move it over

Reply:
It's def a gray area, but ultimately makes it easier for Jorge to keep track of.